src/web/Html.tsx

```diff
@@ -174,6 +176,10 @@ export const Html = React.forwardRef(
       }
     }, [target, transform])
 
+    React.useEffect(() => {
```

Comment:
this should imo be useLayoutEffect or else it comes a frame too late.

Reply:
Thanks, I updated it - I hope that I used the correct git commit syntax